test/hotspot/jtreg/serviceability/dcmd/vm/SystemMapTestBase.java

```diff
@@ -188,7 +189,8 @@ private static class MacOSPatterns implements MapPatterns {
             // java launcher
             macOSbase + macow + space + someNumber + space + "/.*/bin/java",
             // we should see the hs-perf data file, and it should appear as shared as well as committed
-            macOSbase + macprivate + space + someNumber + space + ".*/.*/hsperfdata_.*"
+            // in the jtreg tests this segment appears private
```

Comment:
Since you have two sentences now,  you should capitalize and use ending periods.

Reply:
Done.  Thanks for taking a look.